hw10/main.js

```diff
@@ -0,0 +1,63 @@
+var score = 0;
+document.getElementById("points").innerHTML  = score;
+
+const cards = document.querySelectorAll('.card');
+
+let hasFlippedCard = false;
+let lockBoard = false;
+let firstCard, secondCard;
+
+//hlavní funkce
+function flipCard() {
+  if (lockBoard) return;
+  if (this === firstCard) return;
+
+  this.classList.add('flip');
+
+  if (!hasFlippedCard) {
+    hasFlippedCard = true;
+    firstCard = this;
+    return;
+  }
+  hasFlippedCard = false;
+  secondCard = this;
+
+  checkMatch();
+}
+
+//Kontrolování karet
+function checkMatch() {
+  if (firstCard.dataset.framework === secondCard.dataset.framework) {
+    disable();
+  } else {
+    unflip();
+  }
+
+};
+//správný match
+function disable() {
+  firstCard.removeEventListener('click', flipCard);
+  secondCard.removeEventListener('click', flipCard);
+  score += 1;
+  document.getElementById("points").innerHTML  = score;

```

Comment:
vyber elementu z DOMu provedte pouze jednou a ulozte ho do promenne

Reply:
Dobrý den, nejsem si úplně jistá, jak toto funguje, jelikož to je vaše první poznámka k mým úkolům. Mám možnost tedy ještě úkol upravit a znovu nahrát, abych splnila domácí úkol? Nebo se jen jedná o informaci pro mě, co je špatně a abych se tomu příště vyvarovala.

Reply:
muzete to nahrat znovu a vytvorit novy pull request podle navodu